### PR TITLE
remove: req.namespace is not used for DSC and DSCI

### PR DESCRIPTION
**Pull request by zdtsw**
ref: https://github.com/opendatahub-io/opendatahub-operator/issues/645

## Description

**update**
new image quay.io/wenzhou/opendatahub-operator-catalog:v2.11.645-99
start up
Screenshot from 2023-11-01 16-29-33

**old**

test when created 2nd

 DSCI instance "rhods-setup2"
Screenshot from 2023-10-24 09-25-17

## How Has This Been Tested?

build image:
quay.io/wenzhou/opendatahub-operator-catalog:v2.10.645-2

## Merge criteria:

- [x] The commits are squashed in a cohesive manner and have meaningful messages.
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: controllers/datasciencecluster/datasciencecluster_controller.go [hunk 1]
```diff
@@ -74,15 +74,15 @@ type DataScienceClusterConfig struct {
 // Reconcile is part of the main kubernetes reconciliation loop which aims to
 // move the current state of the cluster closer to the desired state.
 func (r *DataScienceClusterReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	r.Log.Info("Reconciling DataScienceCluster resources", "Request.Namespace", req.Namespace, "Request.Name", req.Name)
+	r.Log.Info("Reconciling DataScienceCluster resources", "Request.Namespace", "cluster-wised", "Request.Name", req.Name)
```
Comment (VaishnaviHire) [inline controllers/datasciencecluster/datasciencecluster_controller.go:77]: Should this be `cluster-wide` ?

File: controllers/dscinitialization/dscinitialization_controller.go [hunk 2]
```diff
@@ -61,7 +61,7 @@ type DSCInitializationReconciler struct {
 
 // Reconcile contains controller logic specific to DSCInitialization instance updates.
 func (r *DSCInitializationReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	r.Log.Info("Reconciling DSCInitialization.", "DSCInitialization", req.Namespace, "Request.Name", req.Name)
+	r.Log.Info("Reconciling DSCInitialization.", "DSCInitialization", "cluster-wised", "Request.Name", req.Name)
```
Comment (VaishnaviHire) [inline controllers/dscinitialization/dscinitialization_controller.go:64]: Same as above

File: controllers/datasciencecluster/datasciencecluster_controller.go (hunk 1 above)
Comment (zdtsw) [reply controllers/datasciencecluster/datasciencecluster_controller.go:77]: or should just remove it compeletely?

Comment (zdtsw) [reply controllers/datasciencecluster/datasciencecluster_controller.go]: updated

**Pull request merged**